src/CHANGES.txt

```diff
@@ -64,6 +64,7 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Fix coding error in docbook tool only exercised when using python lxml 
     - Recognize two additional GNU compiler header directory options in
       ParseFlags: -iquote and -idirafter.
+    - Fix more re patterns that contain \ but not specified as raw strings
```

Comment:
Can you add the list of tools which had these fixes? That'll make it easier to resolve should the changes cause any issues (unlikely, but we have that info now and reverse engineering it would be more trouble)

Reply:
sure... where do you want that info? CHANGES.txt?

Reply:
> sure... where do you want that info? CHANGES.txt?

Yes